tez-ui/pom.xml

```diff
@@ -374,7 +374,7 @@
             </goals>
             <configuration>
               <nodeVersion>${nodeVersion}</nodeVersion>
-              <yarnVersion>v0.21.3</yarnVersion>
+              <yarnVersion>v1.6.0</yarnVersion>
```

Comment:
Please mention the RFC which adds selective dependency resolution in the description.